geopandas/plotting.py

```diff
@@ -736,6 +744,7 @@ def plot_dataframe(
 
     # plot all Polygons and all MultiPolygon components in the same collection
     polys = expl_series[poly_idx & np.invert(nan_idx)]
+    print(polys)
```

Comment:
```suggestion
```

Reply:
my bad ! I had been playing around with the code and forgot to clean up it seems